ubuntu22.04/nvidia-driver

```diff
@@ -303,11 +303,30 @@ _get_module_params() {
     fi
 }
 
-# Load the kernel modules and start persistenced.
-_load_driver() {
+_create_module_params_drop-in() {
     echo "Parsing kernel module parameters..."
     _get_module_params
 
+    if [ ${#NVIDIA_MODULE_PARAMS[@]} -gt 0 ]; then
+        echo "Configuring nvidia module parameters in /etc/modprobe.d/nvidia.conf"
+        echo "options nvidia ${NVIDIA_MODULE_PARAMS[@]}" > /etc/modprobe.d/nvidia.conf
+    fi
+    if [ ${#NVIDIA_UVM_MODULE_PARAMS[@]} -gt 0 ]; then
+        echo "Configuring nvidia-uvm module parameters in /etc/modprobe.d/nvidia-uvm.conf"
```

Comment:
Can we move `/etc/modprobe.d` to a constant var?

Reply:
Sure, updated.